### ISSUE TITLE
add a display for draw force in the "shape" view

### ISSUE THREAD

**Issue by lleroy** [bow-simulation/virtualbow#252]
for extra coolness, as an analog gauge ;-)

**Comment by lleroy** [comment]
for extra coolness, as an analog gage :-)

**Comment by stfnp** [comment]
Good idea! Maybe a text label inside the plot would do the trick, even though not quite as cool as an analog gauge...
